dev/how-to/upgrade/from-previous-version.md

```diff
@@ -160,6 +160,22 @@ If you have previously customized the configuration file of TiDB cluster compone
 
     Recommended configuration: `capacity` = MEM_TOTAL \* 0.5 / the number of TiKV instances.
 
+- In the TiKV configuration, for the multiple instances on a single machine scenario, you also need to configure the `tikv_status_port` parameter.
+
+    ```
+        [tikv_servers]
+        TiKV1-1 ansible_host=172.16.10.4 deploy_dir=/data1/deploy tikv_port=20171 tikv_status_port=20181 labels="host=tikv1"
+        TiKV1-2 ansible_host=172.16.10.4 deploy_dir=/data2/deploy tikv_port=20172 tikv_status_port=20182 labels="host=tikv1"
+        TiKV2-1 ansible_host=172.16.10.5 deploy_dir=/data1/deploy tikv_port=20171 tikv_status_port=20181 labels="host=tikv2"
+        TiKV2-2 ansible_host=172.16.10.5 deploy_dir=/data2/deploy tikv_port=20172 tikv_status_port=20182 labels="host=tikv2"
+        TiKV3-1 ansible_host=172.16.10.6 deploy_dir=/data1/deploy tikv_port=20171 tikv_status_port=20181 labels="host=tikv3"
+        TiKV3-2 ansible_host=172.16.10.6 deploy_dir=/data2/deploy tikv_port=20172 tikv_status_port=20182 labels="host=tikv3"
+    ```
+
+    > **Note:**
+    >
+    > If you upgrade to TiDB 3.0 with a cluster topology of multiple TiKV instances (processes) on a single machine, configure the `tikv_status_port` parameter. Before configuring the parameter, check for port conflicts.
+
 ## Step 4: Download TiDB 3.0 binary to the Control Machine
 
 Make sure that `tidb_version = v3.0.0` in the `tidb-ansible/inventory.ini` file, and then run the following command to download TiDB 3.0 binary to the Control Machine:
```

Comment:
I think the `tidb_version = v3.0.0` should be updated.

Reply:
agree.